### ISSUE TITLE
WiFi password

### ISSUE THREAD

**Issue by Norge100** [MrCrayfish/MrCrayfishDeviceMod#68]
Other pepole can connecting my network, but they can use my printer, so i suggest to add a WiFi password.

**Comment by AtomicAstronaut** [comment]
This is a cool idea!

Hope it gets added, @Norge100.

**Comment by noahball** [comment]
Yes, this would be awesome. I was really hoping this was going to be added in during the development of the router!
